nasa/ow_simulator
# Oceanwater 884 Fix regolith models persist after discard

## Linked Issues:
| EPIC ⚡ | n/a |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-884](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-884) |
| Github :octocat:  | # |


## Summary of Changes
* The end of a discard action now triggers the removal of all regolith particles, just like it does for deliver.

## Test
1. Start sim in any world
2. Run a grind action
3. Call a dig action. Make sure particles have been spawned and collected by the scoop
4. Call the discard action. When scoop movement ceases, make sure all regolith particles have been removed.
